modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/exportimport/data/handler/FileShortcutStagedModelDataHandler.java

```diff
@@ -196,9 +196,15 @@ protected void doImportStagedModel(
 
 		if (portletDataContext.isDataStrategyMirror()) {
 			FileShortcut existingFileShortcut =
-				fetchStagedModelByUuidAndGroupId(
```

Comment:
Hi @balazssk, do we need to change this? It should still work

Reply:
yeah, I think this is more like enforcing a pattern of ERC lookup > UUID lookup in StagedModelDataHandlers

Reply:
I'm unsure about this 🙁. We are changing all SMDHs?

Reply:
[LayoutUtilityPageEntry](https://github.com/balazssk/liferay-portal/blob/master/modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/exportimport/data/handler/LayoutUtilityPageEntryStagedModelDataHandler.java#L112) has something similar